ttnn/cpp/ttnn/operations/moreh/moreh_sum/device/moreh_int_sum_w_program_factory.cpp

```diff
@@ -0,0 +1,225 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <vector>
+
+#include "moreh_sum_device_operation.hpp"
+#include "ttnn/deprecated/tt_dnn/op_library/moreh_helper_functions.hpp"
+#include "tt_metal/common/work_split.hpp"
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
+
+namespace ttnn::operations::moreh::moreh_sum {
+MorehSumOperation::MorehSumWIntFactory::cached_program_t MorehSumOperation::MorehSumWIntFactory::create(
+    const operation_attributes_t& operation_attributes,
+    const tensor_args_t& tensor_args,
+    tensor_return_value_t& output_tensor) {
+    auto input = tensor_args.input;
+    auto output = output_tensor;
+
+    auto output_mem_config = operation_attributes.output_mem_config;
+    const DeviceComputeKernelConfig& compute_kernel_config = init_device_compute_kernel_config(
+        input.device()->arch(), operation_attributes.compute_kernel_config, MathFidelity::HiFi4);
+    ;
+
+    tt::tt_metal::Device* device{input.device()};
+    tt::tt_metal::Program program{tt::tt_metal::CreateProgram()};
```

Comment:
```suggestion
    using namespace tt;
    using namespace tt::tt_metal;
    
    Device* device{input.device()};
    Program program{CreateProgram()};
```
This part doesn't need to be modified right now, but later on, adding using namespace tt; could reduce the amount of code.
example: https://github.com/tenstorrent/tt-metal/blob/main/ttnn/cpp/ttnn/operations/examples/example/device/single_core_program_factory.cpp#L13

Reply:
I'll keep that in mind if we modify them later